### PR TITLE
Refer to VDAF instead of "PPM scheme"

### PR DESCRIPTION
**Pull request by cjpatton**
The spec vaguely refers to the "concrete PPM scheme" whenever we need to talk about protocol specific parameters. This change replaces each such reference with a reference to VDAF. It also adds the aggregation parameter to the aggregate request, which will be needed for hits.

This is an alternative #154 that aims to be less invasive. Partially addresses #98.

### CODE REVIEW

**Review by chris-wood — APPROVED** [review]
Much cleaner 👍

File: draft-gpew-priv-ppm.md
Comment (tgeoghegan) [inline draft-gpew-priv-ppm.md:686]:
```suggestion
  opaque agg_param<0..2^16-1>;  // VDAF aggregation parameter
```

**Review by tgeoghegan — APPROVED** [review]
LGTM, modulo one typo

File: draft-gpew-priv-ppm.md
Comment (ekr) [inline draft-gpew-priv-ppm.md:199]: I think you need to say something more about "p" here. Like, what is it?

And why can't we just bury that in F. I.e., \mathcal{F} is a family of functions parametrized by p and F is a specific instance?

File: draft-gpew-priv-ppm.md
Comment (ekr) [inline draft-gpew-priv-ppm.md:202]:
```suggestion
schemes that implement the VDAF
```

File: draft-gpew-priv-ppm.md
Comment (ekr) [inline draft-gpew-priv-ppm.md:297]: Can we remove the semicolons here?

File: draft-gpew-priv-ppm.md
Comment (ekr) [inline draft-gpew-priv-ppm.md:330]:
```suggestion
the input's validity [BBCGGI19] which the aggregators can jointly verify
```

File: draft-gpew-priv-ppm.md
Comment (ekr) [inline draft-gpew-priv-ppm.md:331]:
```suggestion
reject the report if it cannot be verified. However, they do not
```

File: draft-gpew-priv-ppm.md
Comment (ekr) [inline draft-gpew-priv-ppm.md:734]:
```suggestion
encodes the timestamp, a nonce, and a VDAF-specific `message`:
```

**Review by ekr — APPROVED** [review]
This is generally good.

@cjpatton can you somehow the F(p, ...) thing before you land?

File: draft-gpew-priv-ppm.md
Comment (cjpatton) [reply draft-gpew-priv-ppm.md]: Took this but merged "timestamp" into "nonce".

File: draft-gpew-priv-ppm.md
Comment (cjpatton) [reply draft-gpew-priv-ppm.md]: Replaced with periods.

File: draft-gpew-priv-ppm.md
Comment (cjpatton) [reply draft-gpew-priv-ppm.md:199]: Two reasons not to bury this:
1. `F` is implied by the VDAF in use
2. `p` is a parameter chosen by the collector and distributed to the aggregators.

I agree we should say more about what this is. I've added a paragraph to add a bit more color.

By the way, the canonical example of an aggregation parameter is the set of candidate prefixes for each round of heavy hitters.

Comment (ekr) [reply draft-gpew-priv-ppm.md:199]: I can live with your text here, but I don't really agree with this rationale. This is introductory text and doesn't need to map 1:1 with the VDAF spec.

**Review by ekr — APPROVED** [review]

File: draft-gpew-priv-ppm.md
Comment (cjpatton) [reply draft-gpew-priv-ppm.md:199]: Yeah I agree. Happy to iterate on it later on.

**Comment by cjpatton** [comment]
Squashed.

**Pull request merged**
